server/public/styles.css

```diff
@@ -18,4 +22,5 @@ color: #128781;
 background-color: #EDEDEC;
 font-family:Montserrat;
```

Comment:
I think you need to add the font to the HTML doc to use `Montserrat` . and I believe that the font should be in single or double quotes"